tests/gcoap_dns/app.config

```diff
@@ -0,0 +1,5 @@
+CONFIG_KCONFIG_USEMODULE_GCOAP=y
+CONFIG_GCOAP_PDU_BUF_SIZE=256
+CONFIG_GCOAP_RESEND_BUFS_MAX=2
```

Comment:
No objections, but why?

Reply:
Not sure... I suspect I just copied that config from the `gcoap`/`gcoap_dtls` examples.

Reply:
I just remove that option.